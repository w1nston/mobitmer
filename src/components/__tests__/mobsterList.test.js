import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MobsterList from '../mobsterList';

describe('<MobsterList/>', () => {
  test('renders', () => {
    const component = shallow(<MobsterList />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
