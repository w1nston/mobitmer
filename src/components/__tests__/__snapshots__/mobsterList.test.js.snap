exports[`<MobsterList/> renders 1`] = `
<table>
  <thead>
    <tr>
      <th>
        Mobsters
      </th>
    </tr>
  </thead>
  <tbody />
</table>
`;
